.github/workflows/dependency-review.yaml

```
@@ -10,3 +10,5 @@ jobs:
     permissions:
       contents: read
       pull-requests: write
+    with:
+      base-ref: master
```

```suggestion
      base-ref: github.event.pull_request.base.sha || 'master'
```

`master` alone will not work well for PRs that may be based on a different commit. However, `master` is needed for merge queue.